.github/dependabot.yml

```diff
@@ -0,0 +1,9 @@
+version: 2
+updates:
+  - package-ecosystem: "gomod"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+    ignore:
+      - dependency-name: "staticcheck"
+        versions: [">=0.4.7"] # Ignore versions above 0.4.7, as it requires Go 1.22
```

Comment:
```suggestion
```

This isn't necessary as `staticcheck` is not listed in `go.mod` so Dependabot won't try to update it.